lib/playerlist.js

```diff
@@ -85,54 +109,9 @@ class PlayerList extends LitElement {
       }
     })
 
-    function addPlayer (player) {
-      const playerEntry = document.createElement('div') // There is likely a better way to handle this CSS-wise
-
-      const playerPingContainer = document.createElement('div')
-      const playerPingValue = document.createElement('p')
-      const playerPingLabel = document.createElement('p')
-
-      playerEntry.classList.add('playerlist-entry')
-      playerEntry.innerHTML = player.username
-      playerEntry.id = `plist-player-${player.uuid}`
-
-      if (player.uuid === bot.player.uuid) {
-        playerEntry.classList.add('plist-active-player')
-      }
-
-      playerPingContainer.classList.add('plist-ping-container')
-
-      playerPingValue.classList.add('plist-ping-value')
-      playerPingValue.innerHTML = player.ping
-
-      playerPingLabel.classList.add('plist-ping-label')
-      playerPingLabel.innerHTML = 'ms'
-
-      playerPingContainer.appendChild(playerPingValue)
-      playerPingContainer.appendChild(playerPingLabel)
-      playerEntry.appendChild(playerPingContainer)
-      playerList.appendChild(playerEntry)
-
-      return playerEntry
-    }
-
-    function removePlayer (player) {
-      playerList.querySelector(`#plist-player-${player.uuid}`)?.remove()
-    }
-
-    bot.on('playerUpdated', player => {
-      let playerEntry = playerList.querySelector(`#plist-player-${player.uuid}`)
-
-      if (!playerEntry) {
-        playerEntry = addPlayer(player)
-      }
-
-      playerEntry.querySelector('.plist-ping-value').innerHTML = player.ping
-    })
-
-    bot.on('playerJoined', addPlayer)
-
-    bot.on('playerLeft', removePlayer)
+    bot.on('playerUpdated', () => this.requestUpdate()) // LitElement seems to be batching requests, so it should be fine?
+    bot.on('playerJoined', () => this.requestUpdate())
+    bot.on('playerLeft', (player) => () => this.requestUpdate())
```

Comment:
are you sure this is correct to use double fat arrows? idk seems odd

Reply:
o_O Nope, overlooked that